### ISSUE TITLE
Can't access the plugin website

### ISSUE THREAD

**Issue by Arkuyo** [StephenOTT/Cammand#61]
Sorry for bothering you.
I follow the step in docker folder(README.md), and run from the docker folder: **docker-compose up**.
![image](https://user-images.githubusercontent.com/37679089/124848285-24968280-dfcf-11eb-9016-9e73b0f1138f.png)
But when I reach the site(https://locahost:8080), still see the original camunda website appearance . 
Even though log in, still the same.
![image](https://user-images.githubusercontent.com/37679089/124848335-3841e900-dfcf-11eb-8b63-d9ee8f20fadb.png)
I am not sure which important step I'm missing...
I'm not sure what is really inside the docker/cammand folder is correct.

Please help.
Much appreciate!!!

**Comment by StephenOTT** [comment]
Did you setup the https for the Camunda endpoint? 

There will also always be an error related to the openId conf as it is not configured in the dev demo.

**Comment by Arkuyo** [comment]
Yes, I setup the https by a self-signed certificate with openssl.
And I also add the certificate to the chrome CA trust zone.
Is it work for this?
